inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -490,7 +630,11 @@ bool pass::Serialize::run_on_function(std::shared_ptr<ngraph::Function> f) {
     NGRAPH_CHECK(bin_file, "Can't open bin file: \"" + m_binPath + "\"");
     switch (m_version) {
     case Version::IR_V10:
-        ngfunction_2_irv10(xml_doc, bin_file, *f, m_custom_opsets);
+        {
+            std::string name = "net";
```

Comment:
nitpick: I think we use pass "net" in-place (without local variables).

Reply:
it is problematic for the `XmlSerializer` as its 3rd input `std::string& m_node_type_name` is not const (due to WA for GenericIE op) and "net" name cannot be used  in-place.